aws/required-permissions.md

```diff
@@ -129,3 +131,82 @@ Check out the templates directly for more information.
+| `sns:Publish` | Action | Publish a message to a `ScanResultTopic` created by Cloud One File Storage Security. |
+| `sqs:DeleteMessage` | Action | Delete messages from the `ScannerDLQ` and `PostScanActionDLQ` created by Cloud One File Storage Security. |
+| `sqs:GetQueueAttributes` | Action | Get the attributes of the `ScannerDLQ` and `PostScanActionDLQ` created by Cloud One File Storage Security. |
+| `sqs:ReceiveMessage` | Action | Receive messages from the `ScannerDLQ` and `PostScanActionDLQ` created by Cloud One File Storage Security. |
+| `sqs:SendMessage` | Action | Send scan messages to the `ScannerDLQ` and `PostScanActionDLQ` created by Cloud One File Storage Security. |
+| `ssm:GetParameter` | Action | Retrieve values in AWS Systems Manager that Cloud One File Storage Security created. |
+| `sts:AssumeRole` | Action | Grant the permissions to different principals to switch roles created by Cloud One File Storage Security. |
+| `s3:DeleteObject` | Action | Remove the malicious object from the scanning bucket after being quarantined. |
+| `s3:GetObject` | Action | Retrieve the object to be scanned or quarantined from Amazon S3. |
+| `s3:GetObjectTagging` | Action | List the tags of the scanned object. |
+| `s3:PutObject` | Action | Put the malicious object to quarantine bucket. |
+| `s3:PutObjectTagging` | Action | Put tags on the scanned object. |
+
+### Permissions required by ManagementRole
+
+| Permission | Type | Description |
+| --- | --- | --- |
+| `cloudformation:CreateChangeSet` | Action | Creates a list of changes that will be applied to a stack so that you can review the changes before executing them. |
+| `cloudformation:DescribeStacks` | Action | Allow Cloud One File Storage Security to describe the managed account scanner stack. |
+| `cloudformation:DescribeStackEvents` | Action | Allow Cloud One File Storage Security to describe the events in the managed account scanner stack. |
+| `cloudformation:DescribeStackResource` | Action | Allow Cloud One File Storage Security to describe the deployed resources in the managed account scanner stack. |
+| `cloudformation:DescribeStackResources` | Action | Allow Cloud One File Storage Security to describe the deployed resources in the managed account scanner stack. |
+| `cloudformation:DetectStackDrift` | Action | Allow Cloud One File Storage Security to detect the drift of the managed account scanner stack. |
+| `cloudformation:DetectStackResourceDrift` | Action | Allow Cloud One File Storage Security to detect the drift of the deployed resources in the managed account scanner stack. |
+| `cloudformation:DescribeStackResourceDrifts` | Action | Allow Cloud One File Storage Security to describe the drift of the deployed resources in the managed account scanner stack. |
+| `cloudformation:GetStackPolicy` | Action | Allow Cloud One File Storage Security to get the managed account scanner stack policy. |
```

Comment:
```suggestion
| `cloudformation:GetStackPolicy` | Action | Allows Cloud One File Storage Security to get the managed account scanner stack policy |
```
The third cell in this row contains a clause phrase, not a full sentence, so doesn't require a period. The verb form should be the same for each clause in this column. Because the implied subject (the managed policy) is singular, the verb should be singular which typically ends in "s".